cpp/dolfinx/fem/discreteoperators.h

```diff
@@ -26,75 +26,73 @@ namespace dolfinx::fem
 
 /// @brief Assemble a discrete gradient operator.
 ///
-/// The discrete gradient operator \f$A\f$ takes the gradient of a
-/// Lagrange finite element function \f$v \in V_h \subset H^1\f$ to a
-/// Nedelec (first kind) function \f$w \in W_h \subset H({\rm curl})\f$.
-/// i.e. \f$w=Av\f$ where \f$v\f$ are the Lagrange space
-/// degrees-of-freedom and \f$w\f$ are the curl space
-/// degrees-of-freedom.
-///
+/// The discrete gradient operator \f$A\f$ interpolates take the
```

Comment:
This sentence doesn’t sound quite right anymore.
`…  \f$A\f$ interpolates take the …`